code/modules/cm_marines/overwatch.dm

```diff
@@ -647,15 +651,32 @@
 		if(!M.stat && M.client) //Only living and connected people in our squad
 			if(!only_leader)
 				if(plus_name)
-					M << sound('sound/effects/radiostatic.ogg')
+					M << sound('sound/effects/tech_notification.ogg')
 				to_chat(M, "[icon2html(src, M)] [SPAN_BLUE("<B>Overwatch:</b> [nametext][text]")]")
 			else
 				if(current_squad.squad_leader == M)
 					if(plus_name)
-						M << sound('sound/effects/radiostatic.ogg')
+						M << sound('sound/effects/tech_notification.ogg')
 					to_chat(M, "[icon2html(src, M)] [SPAN_BLUE("<B>SL Overwatch:</b> [nametext][text]")]")
 					return
 
+//Sends a maptext alert to our currently selected squad. Does not make sound.
+/obj/structure/machinery/computer/overwatch/proc/send_maptext_to_squad(var/text = "", var/title_text = "", var/only_leader = 0)
+	if(text == "" || !current_squad || !operator)
+		return //Logic
+
+	var/message_colour = squad_colors_chat[current_squad.color]
+
+	for(var/mob/living/carbon/human/M in current_squad.marines_list)
+		if(!M.stat && M.client) //Only living and connected people in our squad
+			if(!only_leader)
+				M.play_screen_text("<span class='langchat' style=font-size:16pt;text-align:center valign='top'><u>[title_text]</u></span><br>" + text, /obj/screen/text/screen_text/command_order, message_colour)
+				return
+			else
+				if(current_squad.squad_leader == M)
```

Comment:
I feel like this could be done before the for loop. Check if only_leader then check if the current_squad.squad_leader is alive/has a client and play it to the mob and return.

Only move to for loop if not only leader. Hopefully that makes sense.